### ISSUE TITLE
Add vector files for images

### ISSUE THREAD

**Issue by Domsou** [terryworona/TWMessageBarManager#77]
Xcode 6 can handle PDF files to generate images so replace all images by their respective PDF file could save time.

Furthermore by adding vector source files format user can tweaks the image.

**Comment by terryworona** [comment]
The images supplied to the manager are independent of the library itself. 

You can supply any UIImage via the style sheet (pdf, vector, png, etc).

Closing this issue as it is not related to the core library.

**Comment by Domsou** [comment]
I don't understand your answer. So please could you give more precisions ?

My question was about bringing vectors sources to automatically generate all sizes needed because in the current version there are several warning about this particular point. If I am not wrong this is related to the core library.

**Comment by terryworona** [comment]
The core library only cares about a UIImage. 

It's up to the implementor to supply it with whatever it wants. This code is currently in the demo's app delegate, which is outside the core library.

**Comment by Domsou** [comment]
OK. Thank you.
